packages/paste-core/components/skeleton-loader/src/index.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useSpring, animated} from '@twilio-paste/animation-library';
+import {css, styled} from '@twilio-paste/styling-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {LayoutProps, BorderRadiusProps} from '@twilio-paste/style-props';
+
+const AnimatedSkeleton = animated(Box);
+
+const StyledAnimatedSkeleton = styled(AnimatedSkeleton)(() =>
+  css({
+    background: `linear-gradient(90deg, transparent, rgba(255, 255, 255, 0.3) 40%, rgba(255, 255, 255, 0.3) 60%, transparent)`,
+  })
+);
+
+export interface SkeletonLoaderProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    Omit<LayoutProps, 'verticalAlign'>,
+    BorderRadiusProps {
+  disabled?: boolean;
```

Comment:
The more I read the docs, especially when it's mentioned in the accessibility section, leads me to think that the disabled prop is kind of redundant. 

Animation is disabled by reduce motion and doesn't loop more that 5 seconds. So we're really not disabling anything anymore I don't think. We don't have a test for it either which is either a happy co-incidence or we probably don't really care about it enough for it to be useful.

Thought?

Reply:
I think the `disabled` prop is pretty redundant now that we have the animation fine tuned. I'm going to remove it, and if we get feedback about having it, we can always add it back in.